patch/preconfig/packaging-update-abiname.patch

```diff
@@ -9,26 +9,26 @@ From: Olivier Singla <olivier.singla@broadcom.com>
  2 files changed, 2 insertions(+), 2 deletions(-)
 
 diff --git a/debian/config/defines b/debian/config/defines
-index cd4938ecc..e54c10256 100644
+index 983625720..55d7e51a7 100644
 --- a/debian/config/defines
 +++ b/debian/config/defines
 @@ -1,5 +1,5 @@
  [abi]
--abiname: 18
-+abiname: 18-2
+-abiname: 23
++abiname: 23-2
```

Comment:
It’d be great if you elaborated in the commit message, why 23.

Reply:
ABI 23 is the ABI version number that Debian is using. This typically gets incremented by one for each minor kernel update that Debian does (i.e. going from 5.10.179 to the current 5.10.191).

Reply:
Indeed. Please reference the Debian package you base your change on. For example I found [different versions](https://tracker.debian.org/pkg/linux-signed-5.10-arm64): [version 5.10.179+3~deb10u1](https://tracker.debian.org/news/1449695/accepted-linux-signed-510-arm64-5101793deb10u1-source-into-oldoldstable/) and [5.10.179+5~deb10u1](https://tracker.debian.org/news/1452052/accepted-linux-signed-510-arm64-5101795deb10u1-source-into-oldoldstable/), where the change-log of the latter contains:

> Changes:
>  linux-signed-5.10-arm64 (5.10.179+5~deb10u1) buster-security; urgency=high
>  .
>    * Sign kernel from linux-5.10 5.10.179-5~deb10u1
>  .
>    * Rebuild for buster:
>      - Change ABI number to 0.deb10.24

Reply:
I think the versions you found are for Buster (hinted by the `~deb10u1` at the end, which is used for backports).

I've updated the main PR description to include the version number and a link to the publishing of that version.